### ISSUE TITLE
Explain leaf prediction path

### ISSUE THREAD

**Issue by tlapusan** [parrt/dtreeviz#278]
Kind of similar with explain_prediction_path(x).

When looking at the tree structure (or leaf visualisations), you can observe some interesting nodes/leaves and you want to know what characteristics have all the training samples from that node. We can use the split values from the root until the interested node and build some similar output like in explain_prediction_path(x)

**Comment by lchakkei** [comment]
You can use view() to show the leaf prediction path
e.g viz_model.view(x=X_testset[0])
![image](https://user-images.githubusercontent.com/63443381/223721269-963532cb-6f7d-4134-87ef-2f8d02a30104.png)

**Comment by tlapusan** [comment]
Indeed, but it is not convenient/easy to find an x sample which belong to a specific leaf/node. 

what I thought was something like 'explain_node_path(node_id=3)'

**Comment by parrt** [comment]
I guess that makes sense to me except I'm not sure how we know what the node id is. Do we have a way to display that?

**Comment by tlapusan** [comment]
I just put the mouse over a node and little pop-up appear with the node id.
We could include also other info in this way.

<img width="347" alt="Screenshot 2023-03-19 at 11 04 17" src="https://user-images.githubusercontent.com/12815158/226164897-ee33a93a-f99f-413e-bcdb-56f7420972f5.png">

**Comment by parrt** [comment]
cool! Didn't know about that. kind of like the feature earlier we talked about where in leaf display we could pop up node id in bar chart, right?

**Comment by tlapusan** [comment]
yes, could be the same... but it's kind of a hidden functionality. The user has to know it upfront.
